File: docs/2014/snippets/csharp/SQL15/integration services runtime api/application/cs/application_conninfos.cs
//<snippetApplication_connInfosCS>
class ApplicationTests1
    {
        static void Main(string[] args)
        {
          Application app = new Application();

          ConnectionInfos cis = app.ConnectionInfos;
          foreach (ConnectionInfo x in cis)
              Console.WriteLine("Connection Type" + x.ConnectionType);
        }
    }
}

//</snippetApplication_connInfosCS>

File: docs/2014/snippets/visualbasic/SQL15/integration services runtime api/application/vb/application_conninfos.vb
' <snippetApplication_connInfosVB>
class ApplicationTests1
        Sub Main(ByVal args() As String)
          Dim app As Application =  New Application() 
 
          Dim cis As ConnectionInfos =  app.ConnectionInfos 
          For Each x As ConnectionInfo In cis
              Console.WriteLine("Connection Type" + x.ConnectionType)
          Next
        End Sub
End Class
' </snippetApplication_connInfosVB>